src/Interfacer.jl

```diff
@@ -127,4 +128,48 @@ Returns simulation name, if defined, or `Unnamed` if not.
 name(::ComponentModelSimulation) = "Unnamed"
 name(::SurfaceStub) = "SurfaceStub"
 
+"""
```

Comment:
Hmm. These `dss_state` calls contain data structures, like the `integrator`. We don't really want this in the source code, because it's not general - e.g., other models will have their own ways doing dss (or will not need it). It would be better to move all these to the respective init files.

Reply:
Good point! I'll move them to the init files